modules/auxiliary/client/sms/send_text.rb

```diff
@@ -0,0 +1,34 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Auxiliary::Sms
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'SMS Client',
+      'Description'    => %q{
+        This module sends a text message to multiple phones of the same carrier.
+        You can use it to send a malicious link to phones.
+
+        Please note that you do not use this module to send a media file (attachment),
+        because that is MMS.
+      },
+      'Author'         => [ 'sinn3r' ],
+      'License'        => MSF_LICENSE
+    ))
+  end
+
+  def run
+    phone_numbers = datastore['CELLNUMBERS'].split
+    print_status("Sending text (#{datastore['SMSMESSAGE'].length} bytes) to #{phone_numbers.length} number(s)...")
+    res = send_text(phone_numbers, datastore['SMSMESSAGE'])
+    print_status("Done.")
```

Comment:
I think this last bit needs a slight bit of prettying up.
What about putting the send_Text in a try except block.  Doing that, we could then handle things a little nicer.
`print_good("Message sent successfully")` on good conditions

`Net::SMTPAutehnticationError` can be `print_bad("#{datastore['SMTPUSERNAME']}:#{datastore['SMTPPASSWORD']} failed to authenticate")

`Auxiliary triggered a timeout exception` could be re-written to 'Timeout exception, check SMTPADDRESS and SMTPPORT'.

This should give a better result for error checking, and prevent call stack dumps from appearing.